modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -2248,6 +2248,15 @@ private void onCacheStarted(GridCacheContext cacheCtx) throws IgniteCheckedExcep
                 ", mvcc=" + cacheCtx.mvccEnabled() + ']');
         }
 
+        if (ctx.metric().profilingEnabled()) {
```

Comment:
I don't think we need to collect performance statistics about cache start.
It a rare operation that should be analyzed via log files.

Reply:
Excluded.